db/migrate/20160107182205_add_bourbons_user_constraint.rb

```diff
@@ -1,5 +1,9 @@
 class AddBourbonsUserConstraint < ActiveRecord::Migration
-  def change
+  def up
     change_column :bourbons, :user_id, :integer, null: false
   end
+
+  def down
+    change_column :bourbons, :user_id, :integer
+  end
```

Comment:
we wanted to add a null: false to the user_id on the bourbons table. I ran a migration to add the null: false, but it broke on rollback because we had violations to the newly created validation.